lib/utils.ts

```diff
@@ -202,7 +202,7 @@ export function sortBy<T>(array: T[], callback: (a: T) => Comparable): T[];
  * this to sort numbers.
  */
 export function sortBy<T extends Comparable>(array: T[]): T[];
-export function sortBy<T>(array: T[], callback?: (a: T) => Comparable) {
+export function sortBy<T>(array: T[], callback?: (a: T) => Comparable): T[] {
```

Comment:
I'd call this pointless.

Reply:
PS convention is to leave these out. We can discuss changing that convention if you really care, but definitely does not belong here.

```suggestion
export function sortBy<T>(array: T[], callback?: (a: T) => Comparable) {
```